Test/ElasticScale.ShardManagement.UnitTests/ScenarioTests.cs

```diff
@@ -1237,6 +1237,119 @@ public void BasicScenarioRangeShardMaps()
             Assert.IsTrue(success);
         }
 
+        [TestMethod()]
+        [TestCategory("ExcludeFromGatedCheckin")]
+        public void ListShardMapPerformanceCounterValidation()
+        {
+            if (PerfCounterInstance.HasCreatePerformanceCategoryPermissions())
```

Comment:
This makes me nervous. It could be easy for this test to silently succeed and then nobody would ever find a bug. Can you do something like return warning/undecided/etc if the user doesn't have permission?